### ISSUE TITLE
Regular repainting of editor window (how to prevent this?)

### ISSUE THREAD

**Issue by bernd-wechner** [LibVNC/x11vnc#73]
This certainly wasn't happening under vino. And x11vnc is so rich in configurations it's alas both inspiring and disorienting. The specific problem I am seeing is that I am using the TightVNC viewer on a Windows 10 box to connect to and x11vnc server I run manually on an ssh session on a remote Linux Mint 19 box. 

All works fine. 

Oddly though when I work with the Eclipse IDE and edit a file, the whole editor pane seems to repaint every time I click the mouse in that pane. Slowly at that and obviously. 

That's really making it hard to do anything in that editor. So I'm curious, if:

1) there are any well known reasons for this that can be fixed by using appropriate options.
2) there are useful debugging methods for working out why it happens. For example is Eclipse repainting its window? Or x11vnc repainting it on my client?

I have a gut feel it's x11vnc doing the repaint not eclipse. That vino doesn't, and that there's something to do with efforts at cleverly repainting only parts of the window that are playing up because of some subtle screen change when clicking. The selection changes, and so perhaps it's invalidating the whole editor pane? Even so it's really slow, slower than working with vino was. Though x11vnc is much faster in other contexts it seems (that and it's rich configurations bring me here).

**Comment by bk138** [comment]
You're right, there are _lots_ of options. A few I think could be related are:`-wireframe`, `-noxdamage`, `-scrollcopyrect`. Just grep for "repaint" in the README.

**Comment by bernd-wechner** [comment]
Turns out this is Eclipse not x11vnc! This is so beyond the pale I failed to suspect it early enough. But Eclipse is doing this even on the desktop proper sadly. Turns out to have been incomplete diagnostics on my part and so I'll close this.

For the record only, how I found myself here is sort of like this:

- Moved from Mint 18 to Mint 19
- Approved a pile of upgrades in process one of which I'd forgotten was Eclipse Neon to Photon.
- Lost vino in process (it was packaged with Mint 18 and not with Mint 19)
- Looking into vino I remembered everything I hated about it (lousy docs, lousy configs, lousy logging, all sort of like some weekend hack on the gnome project alas), so I looked into alternatives.
- Found x11vnc and was impressed. Tick! Loving it, the huge and clear focus on options on clear detailed logging and more ...

Turns out the Eclipse Neon to Photon upgrade I forgot about has done something that I now need to diagnose (oh joy of joys) and it was completely unrelated to x11vnc. 

So, my apologies for a prematurely filed issue here I think.
